apps/test-suite/tests/Image.js

```diff
@@ -8,7 +8,9 @@ import { mountAndWaitFor, mountAndWaitForWithTimeout, TimeoutError } from './hel
 
 export const name = 'Image';
 
-const REMOTE_SOURCE = { uri: 'https://source.unsplash.com/random' };
+const REMOTE_SOURCE = {
+  uri: 'https://images.unsplash.com/photo-1701743805362-86796f50a0c2?w=1080',
+};
```

Comment:
I know that `/random` endpoint is not working anymore, however now this object is basically the same as `REMOTE_KNOWN_SOURCE` – so maybe a better solution would be to merge it into `REMOTE_SOURCE`?